frontend/catalyst/api_extensions/quantum_operators.py

```diff
@@ -343,47 +339,8 @@ def cond_fn():
             f"to the lenght of control ({len(control)})"
         )
 
-    def _call_handler(*args, _callee: Callable, **kwargs):
-        EvaluationContext.check_is_quantum_tracing(
-            "catalyst.ctrl can only be used from within a qml.qnode."
-        )
-        in_classical_tracers, _ = tree_flatten((args, kwargs))
-        quantum_tape = QuantumTape()
-        with QueuingManager.stop_recording(), quantum_tape:
-            res = _callee(*args, **kwargs)
-        out_classical_tracers, _ = tree_flatten(res)
-
-        _check_no_measurements(quantum_tape)
-
-        region = HybridOpRegion(None, quantum_tape, [], [])
-
-        # Return the operation instance since PL expects this for qml.ctrl(op).
-        return QCtrl(
-            control_wires=control,
-            control_values=control_values,
-            work_wires=work_wires,
-            in_classical_tracers=in_classical_tracers,
-            out_classical_tracers=out_classical_tracers,
-            regions=[region],
-        )
-
-    if isinstance(f, Callable):
-
-        def _callable(*args, **kwargs):
-            return _call_handler(*args, _callee=f, **kwargs)
-
-        return _callable
-
-    elif isinstance(f, Operator):
-        QueuingManager.remove(f)
-
-        def _callee():
-            QueuingManager.append(f)
-
-        return _call_handler(_callee=_callee)
-
-    else:
-        raise ValueError(f"Expected a callable or a qml.Operator, not {f}")  # pragma: no cover
+    res = QCtrlCallable(op, control, control_values=control_values, work_wires=work_wires)
```

Comment:
Nice, I see you already renamed the other classes to better match the PL names (Controlled and HybridControlled), could we also eliminate all other references to "QCtrl"? This class could just CtrlCallable or ControlCallable.
To be fair HybridControlled does sound a bit weird, maybe HybridControl would be sufficient?

Reply:
Does Hybrid make sense here though? Are we using classical instructions somehow? Naming is hard :sweat_smile:

Reply:
I use `CtrlCallable` and it eliminates the ambiguity between Controlled and Control ;)

Reply:
We are in fact allowing classical instructions in the body of the control operator :) The main differentiator to a quantum only version which could hold a nested tape would be that this op contains logic to handle a nested jaxpr scope. 
Naming is definitely hard 😅